library/core/src/ptr/alignment.rs

```diff
@@ -1,14 +1,12 @@
 #![allow(clippy::enum_clike_unportable_variant)]
 
+#[cfg(feature = "ferrocene_certified")]
+use crate::mem;
```

Comment:
non-blocking nit: would be nice to move this to line 9 so it's more clear it's just a subset of the upstream import.

Reply:
rustfmt::skip probably does the trick...